esx_service/cli/vmdkops_admin.py

```diff
@@ -839,6 +839,9 @@ def generate_tenant_ls_rows(tenant_list):
         else:
             default_datastore = vmdk_utils.get_datastore_name(tenant.default_datastore_url)
 
+        if default_datastore is None:
+            return rows
```

Comment:
If this happens for one tenant, the function returns. rows could be half populated. Shouldn't rows be flushed; or you should still traverse remaining tenants in tenant_list if this happens in the middle of iterating  tenant_list?